app/controllers/pto_requests_controller.rb

```diff
@@ -126,12 +171,11 @@ def destroy
             if @pto_request.reason == 'make up / sick day'
                 return sub_make_up_day
             end 
-
             remove_request_info
             RequestsMailer.with(user: @user, pto_request: @pto_request).delete_request_email.deliver_now
             return redirect_to root_path, notice: "Your request was deleted"
         else
-            return redirect_to root_path, notice: "Somthing went wrong"
+            return redirect_to root_path, notice: "Something went wrong"
```

Comment:
Plz to make bad things 'alert:' instead of 'notice:' Good things can stay 'notice:'

Reply:
should be resolved by 05f8385